soc/espressif/esp32/memory.h

```diff
@@ -5,18 +5,19 @@
 #pragma once
 
 /* SRAM0 (192kB)  instruction cache+memory */
-#define SRAM0_IRAM_START    0x40070000
-#define SRAM0_CACHE_SIZE    0x10000
-#define SRAM0_SIZE          0x30000
+#define SRAM0_START           0x40070000
```

Comment:
just wondered if these settings are following the concept of PR #80340

Reply:
No, those need to be reworked if/when #80340 will be merged.